resources/views/components/view.blade.php

```diff
@@ -87,6 +87,9 @@ class="table table-striped snipe-table"
                   <th data-searchable="false" data-sortable="false" data-field="qty">
                     {{ trans('general.qty') }}
                   </th>
+                  <th data-searchable="false" data-sortable="false" data-field="note">
```

Comment:
Is this `note` or `notes`?

Reply:
`note`, since we're pulling in the pivot info